framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
For me that doesn't make much sense, because both would be executed then. Sound like a bit of chaos. Additionally, they're executable even if they're disabled (i.e. by disabling the outer container).

It is not fixable without loosing the enable flag of the Command - or introducing huge changes containing an effective "computedEnabled" property in all Widget instances.